index.js

```diff
@@ -39,4 +49,4 @@ const bumpLoop = () => {
 
 setInterval(bumpLoop, 1000); // Nasty nasty nasty
 
-client.login();
+client.login();
```

Comment:
Files should end with a newline ("officially" a line is defined as having an en of line, some editors may ignore the last line otherwise)